compiler/core/lam_compile_primitive.ml

```diff
@@ -56,14 +56,14 @@ let get_module_system () =
 
 let import_of_path path =
   E.call
-    ~info:{arity = Full; call_info = Call_na}
+    ~info:{arity = Full; call_info = Call_na; call_transformed_jsx = false}
     (E.js_global "import")
     [E.str path]
 
 let wrap_then import value =
   let arg = Ident.create "m" in
   E.call
-    ~info:{arity = Full; call_info = Call_na}
+    ~info:{arity = Full; call_info = Call_na; call_transformed_jsx = false}
```

Comment:
Possible cleanup: define some common `info` values to avoid all these repetitions.